src/stan/model/indexing/lvalue.hpp

```diff
@@ -554,11 +551,11 @@ inline void assign(
     const cons_index_list<index_uni,
                           cons_index_list<index_uni, nil_index_list>>& idxs,
     const U& y, const char* name = "ANON", int depth = 0) {
-  const int m = idxs.head_.n_;
-  const int n = idxs.tail_.head_.n_;
-  stan::math::check_range("matrix[uni,uni] assign range", name, x.rows(), m);
-  stan::math::check_range("matrix[uni,uni] assign range", name, x.cols(), n);
-  x.coeffRef(m - 1, n - 1) = y;
+  stan::math::check_range("matrix[uni,uni] assign row", name, x.rows(),
+                          idxs.head_.n_);
+  stan::math::check_range("matrix[uni,uni] assign column", name, x.cols(),
```

Comment:
`rows` -> `cols` another bug?

Reply:
No this one is still checking both row/col I just changed the labels so users know which one is thrown